src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml

```diff
@@ -1,18 +1,260 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="Org.OData.Capabilities.V1" Alias="Capabilities">
+
+  <!-- Conformance Level -->
+
+  <Term Name="ConformanceLevel" Type="Capabilities.ConformanceLevelType" AppliesTo="EntityContainer" />

```

Comment:
It follows up the Core vocabulary:
https://github.com/OData/odata.net/blob/master/src/Microsoft.OData.Edm/Vocabularies/CoreVocabularies.xml
Besides, such xml is just used to build the Csdl model.